components/badge/index.tsx

```diff
@@ -144,7 +144,7 @@ export default class Badge extends React.Component<BadgeProps, any> {
         data-show={!hidden}
         className={scrollNumberCls}
         count={displayCount}
-        displayComponent={this.renderDispayComponent()} // <Badge status="success" count={<Icon type="xxx" />}></Badge>
+        displayComponent={this.getNumberedDisplayCount()} // <Badge status="success" count={<Icon type="xxx" />}></Badge>
```

Comment:
why that change ?
renderDispayComponent !== getNumberedDispayCount

Reply:
It's my fault, I would fix it and recommit it